test/integration_test_helper.rb

```diff
@@ -332,15 +332,16 @@ def select_location_dropdown(location)
   end
 
   def assert_form_tab(label)
-    within('form .nav-tabs') do
+    within('form .nav-tabs', wait: 2 * Capybara.default_max_wait_time) do
       assert page.has_content?(label)
     end
   end
 
   def switch_form_tab(name)
-    within('form .nav-tabs') do
+    within('form .nav-tabs', wait: 2 * Capybara.default_max_wait_time) do
       click_link name
     end
+    wait_for { page.find(:xpath, "//li[@class='active']//a[contains(text(), '#{name}')]").visible? rescue false }
```

Comment:
Since `find` by default waits and only shows visible elements I'd expect this to also work:
```suggestion
    page.find(:xpath, "//li[@class='active']//a[contains(text(), '#{name}')]")
```